flink/StreamingJob.scala

```diff
@@ -0,0 +1,83 @@
+package vv
```

Comment:
Could you rename the package to something related to the workshop? Like `flink-workshop` or the like